### ISSUE TITLE
[1.4] Weird break on comments in ternary operator

### ISSUE THREAD

**Issue by anmonteiro** [prettier/prettier#1892]
Minimal example:
```js
$ cat x.js
ret = __DEV__ ?
  // $FlowFixMe: this type differs according to the env
vm.runInContext(source, ctx)
: vm.runInThisContext(source, scriptOptions);

$ ./node_modules/.bin/prettier x.js
ret = __DEV__
  ? // $FlowFixMe: this type differs according to the env
    vm
      .runInContext(source, ctx)
  : vm.runInThisContext(source, scriptOptions);
```

**Comment by vjeux** [comment]
I just published 1.4.1 which fixes it!

**Comment by anmonteiro** [comment]
Looks great with 1.4.1. Thanks!
